index.html

```diff
@@ -614,10 +614,11 @@ <h3>Proofs</h3>
 specified as an [[XMLSCHEMA11-2]] combined date and time string.
           </dd>
 
-          <dt><dfn class="lint-ignore">expires</dfn></dt>
+          <dt id="defn-proof-expires">expires</dt>
           <dd>
-An OPTIONAL property that conveys the date and time that a proof expires and that, if present, MUST be
-specified as an [[XMLSCHEMA11-2]] combined date and time string.
+An OPTIONAL property that conveys the date and time that a proof expires and
+that, if present, MUST be specified as an [[XMLSCHEMA11-2]] combined date and
+time string.
```

Comment:
```suggestion
The `expires` property is OPTIONAL. If present, it MUST be an [[XMLSCHEMA11-2]]
`dateTimeStamp` string specifying when the proof expires.
```

Reply:
@dlongley ahh perhaps this is related to your comment on https://github.com/w3c/vc-data-model/issues/1262#issuecomment-1698238618

... confusion intensifies : )

I think what we mean to say here is that "ANY proof with created after `revoked` time MUST be considered invalid at *validation* time... as opposed to verification time, where we expect it to succeed, even for "revoked" verification methods?

Reply:
I think it's possible to parameterize "verification time" (and not always presume it's "the current system time"). It's that time that should be used when looking at these properties to determine if the proof verifies or not. We don't have to have things fall to "validation time". In other words, one could ask some verification software to verify a proof at time `X`.